pkgs/top-level/aliases.nix

```diff
@@ -326,6 +326,8 @@ mapAliases ({
   neutral-style = throw "neural-style has been removed, as the upstream project has been abandoned"; # added 2020-03-28
   nfsUtils = nfs-utils;  # added 2014-12-06
   nginxUnstable = nginxMainline; # added 2018-04-25
+  nginxModules.fastcgi-cache-purge = nginxModules.cache-purge; # added 2020.08.16
+  nginxModules.ngx_aws_auth = nginxModules.aws-auth; # added 2020.08.16
```

Comment:
I don't think this works for nested attrsets. At least it does not work for `pythonPackages`.
You could try to add a `throw` statement within `nginxModules` so.

Reply:
With `throw` not worked. Removed from `aliases`.

Reply:
Just to confirm adding a `throw` for the old attribute name in `pkgs/servers/http/nginx/modules.nix` did not work?

Reply:
Thanks, with this variant worked:
```
error: module fastcgi-cache-purge renamed to cache-purge
```